infrastructure/backups/backup.sh

```diff
@@ -334,12 +336,6 @@ if [[ "$OWN_IP" = "$PRODUCTION_IP" || "$OWN_IP" = "$(dig $PRODUCTION_IP +short)"
   rm -r $BACKUP_RAW_FILES_DIR
 fi
 
-# Cleanup any old backups from influx or mongo. Keep previous 7 days of data and all elastic data
-# Elastic snapshots require a random selection of files in the data/backups/elasticsearch/indices
-# folder
-#------------------------------------------------------------------------------------------------
-find $ROOT_PATH/backups/influxdb -mtime +7 -exec rm {} \;
-find $ROOT_PATH/backups/mongo -mtime +7 -exec rm {} \;
-find $ROOT_PATH/backups/minio -mtime +7 -exec rm {} \;
-find $ROOT_PATH/backups/metabase -mtime +7 -exec rm {} \;
-find $ROOT_PATH/backups/vsexport -mtime +7 -exec rm {} \;
+# Wipe out local copies of backups
+#-------------------------------------
+rm -rf $ROOT_PATH/backups
```

Comment:
I wonder how docker containers behave that have volumes mounted inside this directory? So for instance

```
# Configure influxdb
influxdb:
  volumes:
    - '/data/influxdb:/var/lib/influxdb'
    - '/data/backups/influxdb:/data/backups/influxdb'
```

Reply:
Seems to break actually! I noticed backups had failed 

```
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "Suppressed: org.elasticsearch.ElasticsearchException: failed to create blob container",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.common.blobstore.fs.FsBlobStore.blobContainer(FsBlobStore.java:56) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.repositories.blobstore.BlobStoreRepository.indexContainer(BlobStoreRepository.java:1622) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.repositories.blobstore.BlobStoreRepository.lambda$finalizeSnapshot$46(BlobStoreRepository.java:1460) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.action.ActionRunnable$1.doRun(ActionRunnable.java:34) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:777) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.lang.Thread.run(Thread.java:833) [?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "Caused by: java.nio.file.NoSuchFileException: /data/backups/elasticsearch/indices",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:398) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.nio.file.Files.createDirectory(Files.java:700) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.nio.file.Files.createAndCheckIsDirectory(Files.java:807) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.nio.file.Files.createDirectories(Files.java:793) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.common.blobstore.fs.FsBlobStore.buildAndCreate(FsBlobStore.java:68) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat org.elasticsearch.common.blobstore.fs.FsBlobStore.blobContainer(FsBlobStore.java:54) ~[elasticsearch-7.17.0.jar:7.17.0]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\t... 8 more",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "Suppressed: java.nio.file.NoSuchFileException: /data/backups/elasticsearch/snap-n6jOXs18SdWgW9Co1KsVaA.dat",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:484) ~[?:?]",
opencrvs_elasticsearch.1.x21o25yuo3jo@farajaland-prod    | "\tat java.nio.file.Files.newOutputStream(Files.java:228) ~[?:?]",

```

Reply:
I removed this for now as there is another part in the beginning of the code file

```sh
for BACKUP_DIR in $ROOT_PATH/backups/*; do
  if [ -d "$BACKUP_DIR" ]; then
    rm -rf $BACKUP_DIR/*
  fi
done
```

where the directory is cleaned up